boot/bootutil/src/loader.c

```diff
@@ -930,16 +929,19 @@ boot_validated_swap_type(struct boot_loader_state *state,
      */
 
     if (hdr->ih_magic == IMAGE_MAGIC) {
-        vtable_addr = (uint32_t)hdr + hdr->ih_hdr_size;
-        vtable = (uint32_t *)(vtable_addr);
-        reset_addr = vtable[1];
+        rc = flash_area_read(secondary_fa, hdr->ih_hdr_size +
+				 sizeof(uint32_t), &reset_addr,
+				 sizeof(reset_addr));
```

Comment:
MCUboot source expands tabs to four spaces. Why `sizeof(uint32_t)` and then `sizeof(reset_addr)` for the same variable?

Reply:
I think this probably was reviewed in the previous PR which was made for this patch.
I think the reasoning is to show the `offset=(hrd_address + hrd_size + reset_address_offset)` of the address of the [reset address](https://documentation-service.arm.com/static/5ea823e69931941038df1b02?token=)  in the ARM vector table stored in the start of a firmware image on cortex-m processors. While the other shows the read-size of the variable.

I'll expand the tabs.